wycheproofs/generate-wycheproofs/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "generate-wycheproofs"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Maybe adding `publish = false` may help because the crate looks primarily for testing and not meant to be used by other people.